source/contributors/etiquette.md

```diff
@@ -54,45 +51,45 @@ Review and try to reformulate to be as clear as possible.
+was actually much lower than I originally perceived, and I can
+work _with_, instead of _against_, a given change.
 
   * Be forgiving and accept different priorities. The other person is not
 out to get you or attack your work. More often than not, it is one of the
-above (a-d) that are failing, or that the other person prioritize some
-aspect higher than you do. Sometimes, this requires compromises, sometimes
-not and the different priorities can co-exist.
+above (a-d) that is failing, or the other person prioritizes some
+aspect higher than you do. Sometimes, this requires compromises; sometimes
+the different priorities can co-exist.
 
   * Remember that everyone works on Apache projects as a volunteer.
 People have jobs and lives outside of their Apache projects, and may need 
-more time to even read messages on the list or respond.  Be patient.
+more time to read and respond to messages on the list.  Be patient.
 
-**Most communities at Apache consists of level-headed, reasonable people**, who
-have a strong vested interest in their Apache project. This interest, often
-passion, is both the source of tension, but it is also what unites the
+**Most communities at Apache consists of level-headed, reasonable people** who
+have a strong, vested interest in their Apache project. This interest, often
+passion, can be a source of tension, but it is also what unites
 people within the community. It is easy to forget the vast amount of
 agreement that exists, and get upset over relatively small disagreements.
-Ability to put that aside, or downplay the importance, will ensure a
+Ability to put that aside, or downplay the importance of disagreements, will help ensure a
 harmonious project.
 
 **Face-to-Face** is excellent way to eliminate disagreements, but that is often
-not practical. Consider Skype or Google Hangout, just for the social aspect
+not practical. Consider talking things out on Skype, Google Hangout or the project's Slack channel, just for the social aspect
 of being part of this community. It should not be formal, and the
-invitation should go out to everyone, perhaps someone want to make a short
+invitation should go out to everyonek. Perhaps someone will want to make a short
 presentation of what he/she is doing, to have some "structure", but that
-might not be needed either. Once we have a face to the words, and a general
+might not be needed. Once we can add a face and voice to the words we read, and a general
 idea how that person is socially, we are much more capable to interact by
 email.
 
-**Project decisions must always be made on mailing lists**.  
+**Project decisions must always be made on email lists**.  
 It's great to have a synchronous online chat to work through some questions, but 
 be sure to bring the results of the chat - and any proposals for the 
-project community to consider - back to the mailing list.  People who 
+project community to consider - back to the email list.  People who 
```

Comment:
Here, too, my preference is that we stick with the traditional "mailing list" nomenclature.